src/custom/pages/Claim/ClaimNav.tsx

```diff
@@ -28,20 +29,15 @@ export default function ClaimNav({ account, handleChangeAccount }: ClaimNavProps
           )}
         </div>
         <ClaimAccountButtons>
-          {!!account && (account !== activeClaimAccount || activeClaimAccount === '') && (
-            <ButtonSecondary disabled={isAttempting} onClick={() => setActiveClaimAccount(account)}>
-              Switch to connected account
-            </ButtonSecondary>
-          )}
-
-          {/* Hide account changing action on:
-           * last investment step
-           * attempted claim in progress
-           */}
-          {hasActiveAccount && (investFlowStep < 2 || !isAttempting) && (
-            <ButtonSecondary disabled={isAttempting} onClick={handleChangeAccount}>
-              Change account
-            </ButtonSecondary>
+          {allowToChangeAccount && hasActiveAccount ? (
+            <ButtonSecondary onClick={handleChangeAccount}>Change account</ButtonSecondary>
+          ) : (
+            !!account &&
```

Comment:
might as well make this it's on state const to or is this for TS complaining about account?